sympy/vector/integrals.py

```diff
@@ -1,18 +1,18 @@
-from sympy.core.basic import Basic
 from sympy import simplify
+from sympy.core import Basic, diff
 from sympy.matrices import Matrix
-from sympy.vector import CoordSys3D, Vector, ParametricRegion
+from sympy.vector import CoordSys3D, Vector, ParametricRegion, parametric_region
 from sympy.vector.operators import _get_coord_sys_from_expr
 from sympy.integrals import Integral, integrate
-from sympy.core.function import diff
 from sympy.utilities.iterables import topological_sort, default_sort_key
+from sympy.geometry.entity import GeometryEntity
+
 
 class ParametricIntegral(Basic):
     """
     Represents integral of a scalar or vector field
     over a Parametric Region
-
-    Examples
+/bin    Examples
```

Comment:
why `/bin`?

Reply:
I do not know how this crept in. I will remove it.